Ekaterina_Musatova_дз3/Ekaterina_Musatova_дз3/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Ekaterina_Musatova_дз3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double y, x, x2;
+            Console.WriteLine("enter x:");
+            x = double.Parse(Console.ReadLine());
+            x2 = Math.Pow(x, 2);
+            y = (3 + (2 + 1 / (x2 + 1) / (x2 + 2))) / x2 + 3;
+            Console.WriteLine("y ="+ y);
```

Comment:
Знак равенства лучше отделять пробелом от значения
```suggestion
            Console.WriteLine("y = "+ y);
```